src/services/publishing/publishing.ts

```diff
@@ -35,7 +52,9 @@ const CREATOR_CONTRACTS: CreatorContract = {
   unknown: "0x4Bf7E4777a8D1b6EdD5F2d9b8582e2817F0B0953",
 };
 
-export const getTitleEscrowCreator = async (wallet: Wallet): Promise<TitleEscrowCreator> => {
+export const getTitleEscrowCreator = async (
+  wallet: Wallet | providers.JsonRpcSigner
```

Comment:
Are we able to remove `Wallet`?

Reply:
Oh, according to docs, Wallet and JsonRpcSigner are the same level, they have the same parent `Signer`.
https://docs.ethers.io/v5/api/signer/#Signer

Should we use `Signer` instead?

Reply:
Yea, does that work?